src/index.html

```diff
@@ -16,17 +16,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__page">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        autocomplete="on"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar__form search-bar__form--small"
+      action="http://localhost:8080/"
+      data-qa="small"
+    >
+      <input
+        class="search-bar__input search-bar__input--small"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+        autocomplete="on"
+      />
```

Comment:
Remember about correct indentation between parent and child elements. The input element on lines 39-45 should have a 2-space indentation from the form element.